cmd/root.go

```diff
@@ -68,28 +68,4 @@ func init() {
 		"the default environment")
 	must(viper.BindPFlag("environment", rootCmd.PersistentFlags().Lookup("environment")))
 	must(viper.BindEnv("environment", "ENV"))
-
-	// ratiosAccessToken (required by all)
-	rootCmd.PersistentFlags().StringVarP(&ratiosAccessToken, "ratios-token", "t", "",
-		"the ratios service token for this service")
-	must(viper.BindPFlag("ratios-token", rootCmd.PersistentFlags().Lookup("ratios-token")))
-	must(viper.BindEnv("ratios-token", "RATIOS_TOKEN"))
```

Comment:
Will removing these from root command affect the subcommands (such as the rewards microservice), or are they specified in the subcommands that require them?

Reply:
this was only used by paypal before if i recall correctly

Reply:
reverted this. didn't realize that the server was using